ui-tests/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "ipywidgets-ui-tests",
```

Comment:
Wondering whether this new package should be added to the existing `lerna` packages? Then we might be able to do without `ui-tests/package-lock.json` and keep dependencies in a top-level `yarn.lock`.

Or is it better to keep the `ui-tests` package a bit more separated from the rest?

Reply:
@jtpio good point. Initially I tried adding as dependency to top level package.json. I had some build issues then since rest of the project is using `chai` for unit testing and that was causing name clashes with `jest` that Galata uses (names like `it`, `test`, `expect`). Also, having a separate `ui-tests` directory & package decouples things nicely. Only the developers that add UI tests need to install the dependencies (which include browser binaries) and UI tests can work with different jupyterlab versions than ipywidgets is dependent on.

Reply:
I will add some documentation to guide on adding new UI test cases, building and testing locally.

Reply:
Ah right, there can indeed be some clashes (also since galata is built for a specific version of lab).